setup.py

```diff
@@ -37,23 +32,18 @@
          long_description=open(readme).read(),
          platforms='All',
          install_requires= require,
-         packages=['ibm_db_sa'],
+         packages=['pyodbc_sa'],
         entry_points={
          'sqlalchemy.dialects': [
-                     'db2=ibm_db_sa.ibm_db:DB2Dialect_ibm_db',
-                     'db2.ibm_db=ibm_db_sa.ibm_db:DB2Dialect_ibm_db',
-                     'db2.zxjdbc=ibm_db_sa.zxjdbc:DB2Dialect_zxjdbc',
-                     'db2.pyodbc=ibm_db_sa.pyodbc:DB2Dialect_pyodbc',
-                     'db2.zxjdbc400=ibm_db_sa.zxjdbc:AS400Dialect_zxjdbc',
-                     'db2.pyodbc400=ibm_db_sa.pyodbc:AS400Dialect_pyodbc',
+                     'db2=pyodbc_sa.pyodbc:DB2Dialect_pyodbc',
+                     'db2.pyodbc=pyodbc_sa.pyodbc:DB2Dialect_pyodbc',
+                     'db2.pyodbc400=pyodbc_sa.pyodbc:AS400Dialect_pyodbc',
```

Comment:
I think we only need to register one dialect now, since there is only one. Previously they had mulitple dialects for combinations of Db2 platform (LUW, IBM i, and z/OS) as well as backend (ibm_db, pyodbc, zxjdbc). We now only have 1 Db2 platform and 1 backend, so we only need to register that 1 combination.

Reply:
It should also probably be just "ibmi" as well, to match the package name.